helm_deploy/laa-crown-court-proceeding/templates/_environment.tpl

```diff
@@ -37,19 +37,19 @@ env:
   - name: EVIDENCE_API_OAUTH_CLIENT_SECRET
     value: {{ .Values.evidenceApi.clientSecret }}
   - name: CLOUD_PLATFORM_QUEUE_REGION
-    value: {{ .Values.cloud-platform.aws.sqs.region }}
+    value: {{ .Values.cloud_platform.aws.sqs.region }}
   - name: CLOUD_PLATFORM_QUEUE_ACCESS_KEY
-    value: {{ .Values.cloud-platform.aws.sqs.accessKey }}
```

Comment:
[Helm best practices:](https://helm.sh/docs/chart_best_practices/values/) Variable names should begin with a lowercase letter, and words should be separated with camelcase:

Reply:
Much prefer underscore case, but point taken!